src/easydmp/dmpt/models.py

```diff
@@ -295,6 +296,10 @@ def get_summary(self, data, valid_section_ids=()):
                 # 2/2 Otherwise this might edit the actual plan data in memory!
                 # Mutable types strike back..
                 value['question'] = question
+                if answer and question.is_optional_section_question() and answer.get('choice', None) == 'No':
```

Comment:
You could use `question._serialize_condition(answer) == 'No'` instead of `answer.get('choice', None) == 'No'` here. I guess I should remove the prefixed `_` from that method... it's starting to be too useful to hide.